src/content/advanced/security-policy-enforcement/policy-api.md

```diff
@@ -0,0 +1,69 @@
+
+For example, when the Pod Security Policy type was deprecated and removed, many workloads had to be re-evaluated and new exceptions created for them, even though neither the workload nor the risk had changed.
+Some new features were gained, but the effort was primarily invested in order to maintain feature parity and avoid security regressions.
+We expect that Kubernetes and third-party tooling will continue to evolve, and that we can help our customers be faster if they are not directly tied to tool-specific interfaces which they don't actually want to manage.
+
+So, we created the Policy API in order to allow Giant Swarm to more seamlessly and transparently move clusters between policy implementations, and to reduce the overall toil of dealing with common security configuration.
+
+### At a glance
+
+The Policy API:
+
+- is an interface for configuring the various types of (mostly security-related) policies which Giant Swarm manages.
+- provides a way for cluster administrators to declare their intent about which policies to enforce and which resources are exempt from those policies.
+- is intended to manage additional policy types in the future, including networking, vulnerability management, anomaly detection, and others.
+- is _not_ a general purpose policy syntax language. Users cannot define custom policies via the Policy API. It can only be used to configure the policies Giant Swarm actively manages.
+- does _not_ hide the underlying implementations. Users are free to directly use the underlying tools or APIs. The only difference is that _Giant Swarm will not manage, migrate, or adopt any policies or configuration passed directly_.
+- generates native resources for the underlying implementations. These resources continue to function even if the Policy API controllers are removed.
+  
+#### Managed and un-managed policies
+
+The Policy API only orchestrates policies which Giant Swarm actively manages. It does not interfere with customer policies, exceptions, or configuration which are managed externally to the Policy API.
+
+For example, Giant Swarm enforces security policies in every cluster by default, and advises customers to use the Policy API to declare exceptions for any workloads which need them.
+We currently use Kyverno to enforce those policies, and the Policy API generates Kyverno PolicyExceptions based on the exceptions configured through the Policy API.
+
+A cluster administrator might choose to use that managed Kyverno to enforce their own additional policies, to enforce some business-specific logic, for instance.
+They can easily do that by creating a new Kyverno ClusterPolicy, and creating Kyverno PolicyExceptions for any approved exceptions.
+
+If, hypothetically, in the future, Giant Swarm chooses to stop managing Kyverno as part of our standard platform, we would use our Policy API controllers to move our managed policies and any relevant exceptions into a new implementation which maintains the desired behavior.
```

Comment:
See above review comment which suggests the important point – customers don't need to change anything in case of migration. If we apply the above suggestion, this sentence _here_ can go away.

Reply:
Commented above